### PR TITLE
Install Ubuntu

### PR DESCRIPTION
**Pull request by theishshah**
Issue #2

### CODE REVIEW

**Comment by theishshah** [comment]
Travis won't accept bootloader and is throwing me a spelling error @narendasan can you take a look so we can merge/fix it in travis?

File: _til/linux/install-ubuntu.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Do you want to move from Windows into a UNIX like dev environment? Ubuntu is an easy way to get your feet wet and try out Linux.
+
+## Basic Installation
+
+Ubuntu ships with a very straight forward installer and makes set up a snap. Here are the only steps:
+* Choose a language and keyboard layout (Usually English-US for both)
+* Connect to a network (Optional, convenient to install updates during installation)
+* Choose to install updates
+* Choose to install 3rd party non-free items (Flash, MP3, etc. I usually select yes as it makes the user experience much easier)
```
Comment (narendasan) [inline _til/linux/install-ubuntu.md]: Comment here on how free is means free as in libre and not something you have to pay for

File: _til/linux/install-ubuntu.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* Create a user and hostname
```
Comment (narendasan) [inline _til/linux/install-ubuntu.md]: note hostname == name of your computer

File: _til/linux/install-ubuntu.md (hunk 1 above)
Comment (narendasan) [inline _til/linux/install-ubuntu.md]: make a note here that this is free as in libre and not something you need to pay for

File: _til/linux/install-ubuntu.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* Partition the disk (See below)
```
Comment (narendasan) [inline _til/linux/install-ubuntu.md]: Manual disk partitioning is typically not required. There are options to install over any existing OS (which will also apply to no OS) and to install along side the existing OS. This should be recommended unless more control is needed

File: _til/linux/install-ubuntu.md (hunk 2 above)
Comment (narendasan) [inline _til/linux/install-ubuntu.md]: note that the hostname == name of you computer

File: _til/linux/install-ubuntu.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* Choose your location for time/date
+* Sit back, relax, and wait a few minutes for installation to finish
+
+## Partitions
```
Comment (narendasan) [inline _til/linux/install-ubuntu.md]: include:
> Partitioning is required if you want more complicated layouts for you storage i.e you want all your documents on one disk but the os on another, or you are dealing with a volatile system and need to insure exact partition layouts.

File: _til/linux/install-ubuntu.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: Install Ubuntu 
+category: linux
+---
+
+Do you want to move from Windows into a UNIX like dev environment? Ubuntu is an easy way to get your feet wet and try out Linux.
+
```
Comment (narendasan) [inline _til/linux/install-ubuntu.md:7]: Instructions on how to create a boot disk pls

**Review by narendasan — CHANGES_REQUESTED** [review]

**Comment by narendasan** [comment]
@theishshah, can you you address these issues?

**Comment by theishshah** [comment]
@narendasan updated!

**Comment by theishshah** [comment]
@narendasan travis is failing over bootable, iso os dropdown and whatnot. no actual spelling errors

**Comment by narendasan** [comment]
@theishshah  add them to dictionary.yml

File: _til/linux/install-ubuntu.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+## Creating bootable media
```
Comment (narendasan) [inline _til/linux/install-ubuntu.md:8]: Add instructions for Ubuntu (use startup disk creator) and Mac ideally

File: _til/linux/install-ubuntu.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+## Basic Installation
+
+Ubuntu ships with a very straight forward installer and makes set up a snap. Here are the only steps:
+* Choose a language and keyboard layout (Usually English-US for both)
+* Connect to a network (Optional, convenient to install updates during installation)
+* Choose to install updates
+* Choose to install 3rd party non-free items (Free as libre, not cost wise. The non-free items are non-open source, but will cost you nothing)
+* Create a user and hostname (hostname is the name of your computer)
+* Partition the disk (See below. Manual partitioning is generally not required as the installer will offer both dual-boot options and full replacement automatically)
```
Comment (narendasan) [inline _til/linux/install-ubuntu.md:29]: Change this to: Select how to install the OS (either overwrite the disk, install along side another OS or a more complicated layout (LINK here)) 

Prioritize the easy stuff, it makes things more accessible

File: _til/linux/install-ubuntu.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+Home: This partition will store all of the user data. On systems with only one disk and one Linux installation, there isn't always a need to sperate this form the root partition
+
+Root: This partition is the meat and potatoes of the systems. It holds all of the core utilities and applications.
+
+### Recommendation:
+
+I generally use 1.5x my system RAM for swap and the rest of the disk space as root. By default the root partiition will encompass home and boot information, and the default Ubuntu installer will configure the bootloader correctly for most use cases.
+
+## Dual Booting
```
Comment (narendasan) [inline _til/linux/install-ubuntu.md:48]: Make some of this a sub section of partitioning or something and the other about the bootloader

**Review by narendasan — CHANGES_REQUESTED** [review]

**Comment by narendasan** [comment]
@theishshah

**Comment by narendasan** [comment]
@theishshah

**Pull request closed without merging**